shell.nix

```diff
@@ -0,0 +1,54 @@
+{
+  githubPAT ? builtins.getEnv "GITHUB_PAT",
```

Comment:
I was expecting this to work. I was assuming the GITHUB_PAT was supposed to result in skipping this password step? Am I approaching this wrong? `gh auth token` works outside of this command. 
<img width="402" alt="Screenshot 2025-03-20 at 6 31 18 PM" src="https://github.com/user-attachments/assets/ecf8d2dc-7a62-4f25-8c67-5d0e562cca12" />